optax/losses/classification.py

```diff
@@ -430,3 +430,49 @@ def ctc_loss(
       logits, logit_paddings, labels, label_paddings,
       blank_id=blank_id, log_epsilon=log_epsilon)
   return per_seq_loss
+
+
+
+def sigmoid_focal_loss(
+    logits:  chex.Array,
+    labels:  chex.Array,
+    alpha: Optional[float] = None,
+    gamma: float = 2,
+) ->  chex.Array:
+  """Compute a sigmoid focal loss as proposed by Lin et al.
+  The focal loss is a re-weighted cross entropy for unbalanced problems.
+  Use this loss function if classes are not mutually exclusive.
+  See `sigmoid_binary_cross_entropy` for more information.
+
+  References:
+    Lin et al. https://arxiv.org/pdf/1708.02002.pdf
+  
+  Args:
+    logits: A float array of arbitrary shape.
+      The predictions for each example.
+    labels: A float array, its shape must be identical to
+      that of logits. It containes the binary
+      classification label for each element in logits
+      (0 for the out of class and 1 for in class).
+      This array is often one-hot encoded.
```

Comment:
Does this mean that it's often either 0 or 1? If this is true, I think it's a bit confusing since we just said these are logits. I would remove the last phrase since it seems unnecessary. What do you think?

Reply:
It's probably not wise to talk about the logits all the time in the label docstring. [8a85e61](https://github.com/google-deepmind/optax/pull/599/commits/8a85e6112206a77940cdc115cdfa4ebc61027071) attempts to address your concerns by re-writing the docstring. The last phrase is gone now.